RELEASES.md

```diff
@@ -1,5 +1,131 @@
 # Thoth Release Notes
 
+This release notes file describes new and important changes to the frontend services and toolbox and to backend
+services and the knowledge graph of Thoth.
+
+Release cycles may vary from 2 to 4 weeks, depending on complexity of features or capacity of the team! For the current
+state of planning, have a look at out [Projects](https://github.com/orgs/thoth-station/projects).
+
+## Release 2020.12.01
+
+After releasing some features to our backend and knowledge graph last time, we are now happy to announce the avaiability
+of new GitHub App to the general public! The two apps help you keeping your source code fresh and up to data, and
```

Comment:
why are saying kebechet is new ? 
are we trying to say: `it is newly available in public space for helping the general public with dependent updates` 

and the `two apps`  is describe but we haven't spoken about any other app, is this regarding the `reproducible deployment pipeline via aicoe-ci`